nativescript-angular/renderer.ts

```diff
@@ -365,6 +365,11 @@ const addStyleToCss = profile('"renderer".addStyleToCss', function addStyleToCss
     addCss(style);
 });
 
+// tslint:disable-next-line
+const addScopedStyleToCss = profile('"renderer".addStyleToCss', function addScopedStyleToCss(style: string): void {
```

Comment:
```suggestion
const addScopedStyleToCss = profile('"renderer".addScopedStyleToCss', function addScopedStyleToCss(style: string): void {
```

Sorry I didn't notice this before.